Gemfile

```diff
@@ -58,6 +58,15 @@ gem "rack-attack" # rate-limiting
 gem "lograge" # for JSON logging
 gem "silencer" # to disable default logging in prod
 
+# mockturtle
+gem "ircinch"
+gem "json"
+gem "nokogiri"
+gem "open-uri"
```

Comment:
Please remove this and use `extras/sponge.rb` to fetch anything. It prevents people from saying in chat "https://127.0.0.1:22" etc and using the bot to portscan/exploit us or others.